### DISCUSSION TITLE
Request Log Analysis for Efficient Mockup Creation

### DISCUSSION THREAD

**Discussion by thinhngotony** [mockoon/mockoon#1163] [Help]
I'm currently dealing with a large project with many unfamiliar request formats. To save time, I'd like to copy request logs to another environment to see real responses and create mockups. Is there a way to do this efficiently?

**Comment by 255kb** [comment]
I'm not sure to 100% understand what you want to achieve, but could the [recording mode](https://mockoon.com/docs/latest/logging-and-recording/auto-mocking-and-recording/#record-your-api-calls) solve your issue?
